apps/condo/domains/billing/schema/RegisterBillingReceiptsService.js

```diff
@@ -26,10 +27,14 @@ const { BillingReceipt } = require('@condo/domains/billing/utils/serverSchema')
 const {
     BillingIntegrationOrganizationContextForRegisterBillingReceiptsService: BillingContextApi,
 } = require('@condo/domains/billing/utils/serverSchema')
+const { ReceiptInputCache } = require('@condo/domains/billing/utils/serverSchema/receiptInputCache')
 
 const appLogger = getLogger('condo')
 const registerReceiptLogger = appLogger.child({ module: 'register-billing-receipts' })
 
+const redisClient = getRedisClient('register-billing-receipts', 'cache')
```

Comment:
you can use `default` redis client. I don't think that we will move it to another redis server.